### PR TITLE
Added package findtests

### PR DESCRIPTION
**Pull request by jackvandrunen**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -9356,5 +9356,21 @@
...
+  },
+  {
+    "name": "findtests",
+    "url": "https://github.com/jackvandrunen/findtests",
+    "method": "git",
+    "tags": [
+      "test",
+      "tests",
+      "testing",
+      "unit test",
```
Comment (dom96) [inline packages.json]: no spaces in tags please

**Comment by dom96** [comment]
Otherwise looks good

**Comment by jackvandrunen** [comment]
Saw that some other packages had it so thought it was ok. All fixed now!

**Comment by dom96** [comment]
What other packages have it?

**Comment by jackvandrunen** [comment]
From a quick glance it seems like quite a few of the earlier packages have them. I specifically saw them in einheit because I stole some unit-test-related flags, but you can see that there are at least a couple other packages around that area which have the same issue.

**Pull request merged**